### DISCUSSION TITLE
MyAsus critical update not appearing in g-helper updates

### DISCUSSION THREAD

**Discussion by demandred1687** [seerge/g-helper#2438] [Q&A] [answered]
Hi,
is it expected for some drivers to appear in myasus but not in g-helper? for example - 

![image](https://github.com/seerge/g-helper/assets/149145786/6a1e6ff8-df7e-40e6-b7a4-b1cc42de81ad)

thanks and love your work!

**Comment by seerge** [comment, marked as answer]
@demandred1687 hello, G-Helper shows you a mirror of official asus website / api with drivers and BIOS updates. 

It takes them from 
https://rog.asus.com/support/webapi/product/GetPDDrivers?website=global&model=GA402XZ&cpu=GA402XZ&osid=52

That update is not there :) and I don't know from where MyASUS takes that "OpenGL / Vulkan" settings.

But based from it's name I can only imagine that "update" is something relevant if you use Asus own GPU drivers for Nvidia or AMD from 2023 year ... :)

**Reply by demandred1687** [reply]
thanks for clarifying Serge. I wish Asus had them all in one place so it get's picked up by g-helper. Unfortunately for me, it would mean that i would need to keep the asus services enabled from time to time to check for any other driver updates.
